app/src/main/java/org/mozilla/fenix/home/sessioncontrol/SessionControlInteractor.kt

```diff
@@ -383,21 +383,24 @@ class SessionControlInteractor(
     }
 
     override fun onHistoryShowAllClicked() {
-        historyMetadataController.handleHistoryShowAllClicked()
+        recentVisitsController.handleHistoryShowAllClicked()
     }
 
     override fun onRecentHistoryGroupClicked(recentHistoryGroup: RecentHistoryGroup) {
-        historyMetadataController.handleRecentHistoryGroupClicked(
+        recentVisitsController.handleRecentHistoryGroupClicked(
             recentHistoryGroup
         )
     }
 
-    override fun onRemoveRecentHistoryGroup(searchTerm: String) {
-        historyMetadataController.handleRemoveRecentHistoryGroup(searchTerm)
+    override fun onRemoveRecentHistoryGroup(groupTitle: String) {
+        recentVisitsController.handleRemoveRecentHistoryGroup(groupTitle)
     }
 
+    /**
+     *
+     */
```

Comment:
Can probably remove this

Reply:
Started to add a comment but couldn't come up with something better than in the interface :-)).
Will remove, thank you!